examples/manageddisk_lustre/config.json

```diff
@@ -0,0 +1,257 @@
+    "headnode": {
+      "type": "vm",
+      "vm_type": "Standard_HB60rs",
+      "accelerated_networking": false,
+      "public_ip": true,
+      "image": "variables.hpcimage",
+      "subnet": "compute",
+      "tags": [
+        "disable-selinux",
+        "lfsclient",
+        "lfsazimport",
+        "localuser",
+        "pbsserver",
+        "loginnode",
+        "nfsserver"
+      ]
+    },
+    "compute": {
+      "type": "vmss",
+      "vm_type": "Standard_HB60rs",
+      "accelerated_networking": false,
+      "instances": 2,
+      "image": "variables.hpcimage",
+      "subnet": "compute",
+      "tags": [
+        "disable-selinux",
+        "lfsclient",
+        "localuser",
+        "pbsclient",
+        "nfsclient"
+      ]
+    },
+    "lfsmds": {
+      "type": "vm",
+      "vm_type": "Standard_D16s_v3",
+      "accelerated_networking": true,
+      "image": "variables.image",
+      "subnet": "storage",
+      "storage_sku": "Premium_LRS",
+      "data_disks": [
+        4096
+      ],
+      "tags": [
+        "lfspkgs",
+        "lfsmaster",
+        "disable-selinux",
+        "lfsloganalytics"
+      ]
+    },
+    "lfsoss": {
+      "type": "vmss",
+      "vm_type": "Standard_D16s_v3",
+      "instances": 2,
+      "accelerated_networking": true,
+      "image": "variables.image",
+      "subnet": "storage",
+      "storage_sku": "Premium_LRS",
+      "data_disks": [
+        1024,
+        1024,
+        1024,
+        1024,
+        1024,
+        1024
+      ],
+      "tags": [
+        "lfspkgs",
+        "lfsoss",
+        "disable-selinux",
+        "lfsloganalytics"
+      ]
+    },
+    "lfshsm": {
+      "type": "vmss",
+      "vm_type": "Standard_D16s_v3",
+      "instances": 1,
+      "accelerated_networking": true,
+      "image": "variables.image",
+      "subnet": "storage",
+      "tags": [
+        "lfspkgs",
+        "lfshsm",
+        "disable-selinux"
+      ]
+    },
+    "lfssmb": {
+      "type": "vm",
+      "vm_type": "Standard_D16s_v3",
+      "instances": 1,
+      "accelerated_networking": true,
+      "image": "variables.image",
+      "subnet": "storage",
+      "tags": [
+        "lfspkgs",
+        "lfsclient",
+        "lfssmb",
+        "disable-selinux",
+        "localuser"
+      ]
+    }
+  },
+  "install": [
+    {
+      "script": "disable-selinux.sh",
+      "tag": "disable-selinux",
+      "sudo": true
+    },
+    {
+      "script": "nfsserver.sh",
+      "tag": "nfsserver",
+      "sudo": true
+    },
+    {
+      "script": "nfsclient.sh",
+      "args": [
+        "$(<hostlists/tags/nfsserver)"
+      ],
+      "tag": "nfsclient",
+      "sudo": true
+    },
+    {
+      "script": "localuser.sh",
+      "args": [
+        "$(<hostlists/tags/nfsserver)"
+      ],
+      "tag": "localuser",
+      "sudo": true
+    },
+    {
+      "script": "create_raid0.sh",
+      "args": [
+        "/dev/md0",
+        "/dev/sdc /dev/sdd /dev/sde /dev/sdf /dev/sdg /dev/sdh"
```

Comment:
you could try simplifying this is bash expansion by doing something like:

    "/dev/sd[c-l]"

That *should* pick up the first 10 disks (assuming it starts with sdc and is consecutive) so you can change the disks in the config file without needing to change the arguments here.

You would need to test this though.

Reply:
Something will be changed, maybe even auto-detecting the new drives.

Reply:
What do you mean here?  My above suggestion should find up to 10 premium disks that are added (although it needs testing).  If they are not attached, they should not appear in the list.

Reply:
Change made. Code ready for merge, but please discuss Compute Node type with Xavier. Currently using HB servers.